### PR TITLE
Rewrite healthcheck and simplify readme

### PR DESCRIPTION
**Pull request by rafal-gorecki**

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -1,36 +1,53 @@
...
+    sed -i '/find_package(sensor_msgs REQUIRED)/a \
+            add_executable(healthcheck_node src/healthcheck.cpp)\n \
+            ament_target_dependencies(healthcheck_node rclcpp sensor_msgs)\n \
+            install(TARGETS healthcheck_node DESTINATION lib/${PROJECT_NAME})' \
+            /ros2_ws/src/healthcheck_pkg/CMakeLists.txt
+
+COPY ./healthcheck.cpp /ros2_ws/src/healthcheck_pkg/src/
 
+# Build
+RUN source /opt/ros/$ROS_DISTRO/setup.bash && \
```
Comment (delihus) [inline Dockerfile:29]: We can remove `console_direct+`

File: Dockerfile [hunk 2]
```diff
@@ -1,36 +1,53 @@
...
+    colcon build --event-handlers console_direct+
```
Comment (delihus) [inline Dockerfile]:
```suggestion
    colcon build
```

File: README.md [hunk 3]
```diff
@@ -1,71 +1,48 @@
...
-On the second computer (or the first one) with connected display run:
+**2. Specify Configuration**
 
 ```bash
-cd demo
-xhost local:root
-docker compose -f compose.rviz.yaml up
+# Select LIDAR baudrate:
+# - 115200 - for RPLIDAR A2M8 (red circle around the sensor)
+# - 256000 - for RPLIDAR A2M12 and A3 (violet circle around the sensor)
```
Comment (delihus) [inline README.md]: Maybe add info about S1, S2, S3?

**Review by delihus — APPROVED** [review]

File: README.md (hunk 3 above)
Comment (rafal-gorecki) [reply README.md]: Thanks

**Pull request merged**